ibis/backends/pyspark/tests/test_import_export.py

```diff
@@ -1,9 +1,42 @@
 from __future__ import annotations
 
 from operator import methodcaller
+from time import sleep
+from unittest import mock
 
+import pandas as pd
 import pytest
 
+from ibis.backends.conftest import TEST_TABLES
+from ibis.backends.pyspark import Backend
+from ibis.backends.pyspark.datatypes import PySparkSchema
+
+
+@pytest.fixture(scope="session", autouse=True)
+def default_session_fixture():
+    with mock.patch.object(Backend, "write_to_memory", write_to_memory, create=True):
+        yield
+
+
+def write_to_memory(self, expr, table_name):
+    if self.mode == "batch":
+        raise NotImplementedError
+    df = self._session.sql(expr.compile())
+    df.writeStream.format("memory").queryName(table_name).start()
+
+
+@pytest.fixture(autouse=True, scope="function")
+def stop_active_jobs(con_streaming):
+    yield
+    for sq in con_streaming._session.streams.active:
```

Comment:
Any reason not to put this in the `con_streaming` fixture?

`yield` fixtures will only run the code after the `yield` on teardown.

Reply:
(IIRC) it's because we need this to run on a function scope (after every test) rather than session scope